blank-canvas-blocks/sass/blocks/_navigation.scss

```diff
@@ -3,22 +3,15 @@
 	a {
 		border-bottom: none;
```

Comment:
This line caught my attention, even though you didn't add it. I checked the small themes that we're looking to develop and indeed there are is no border bottom to be found, but I'm wondering why we need to do this here.

It looks like we set the default link style (not yet configurable) over here: https://github.com/Automattic/themes/blob/make/blank-canvas-blocks/blank-canvas-blocks/sass/elements/_links.scss#L7

Then we need to override it because the menu should not ever use underlined links? Does that make sense to you?

Reply:
The anchor styles originally came pretty much as-is from Seedlet and that informed the navigation styles I think.  Is this the right opportunity to revisit those default anchor styles?  I believe @kjellr confirmed that the preferred default style (and likely default Gutenberg behavior) is to use the underline style as the primary link defining mechanism.

Seems to me like we should refactor that link style to use underline instead of border bottom and if we want to implement that effect (for instance in Seedlet-blocks) then the customization to do so would be to the responsibility of the theme.

Reply:
Links are refactored to use underline here: https://github.com/Automattic/themes/pull/3478